deb/openmediavault/srv/salt/omv/deploy/postfix/files/main.cf.j2

```diff
@@ -37,8 +37,8 @@ smtp_sasl_security_options = {{ smtp_sasl_security_options }}
 smtp_sasl_password_maps = hash:{{ smtp_sasl_password_maps }}
 smtp_sasl_mechanism_filter = {{ smtp_sasl_mechanism_filter }}
 {%- endif %}
-{%- if config.tls | to_bool %}
-{%- if config.port == 465 %}
+{%- if config.tls == 'ssl' or config.tls == 'starttls' or config.tls == 'auto' %}
+{%- if config.tls == 'starttls' or (config.tls == 'auto' and config.port == 465) %}
```

Comment:
This line is three chars longer than recommended, I hope that's okay.

Reply:
Yes, [no problem](https://github.com/openmediavault/openmediavault/blob/master/.editorconfig#L24) (doing this in Jinja files will result in expected unexpect behaviour :-)).